14-location.Rmd

```diff
@@ -234,32 +230,21 @@ Plotting this reveals eight metropolitan regions (Figure \@ref(fig:metro-areas))
 Each region consists of one or more raster cells.
 It would be nice if we could join all cells belonging to one region.
 **terra**'s\index{terra} `patches()` command does exactly that.
-Subsequently, `as.polygons()` converts the raster object into spatial polygons, and `st_as_sf()` converts it into an `sf`-object.
+Subsequently, `as.polygons()` converts the raster object into spatial polygons, and `st_as_sf()` converts it into an `sf` object.
 
 ```{r 14-location-13, warning=FALSE, message=FALSE}
-polys = pop_agg |> 
+metros = pop_agg |> 
   terra::patches(directions = 8) |>
   terra::as.polygons() |>
   sf::st_as_sf()
 ```
 
-`polys` now features a column named `patches` which indicates to which metropolitan region each polygon belongs and which we will use to dissolve\index{dissolve} the polygons into coherent single regions (see also Section \@ref(geometry-unions)):
```

Comment:
Why have you deleted this part? We need the dissolved regions for the reverse geocoding later on.

Reply:
And again, you are right, the dissolving operation is no longer necessary. Thanks for spotting!

Reply:
Yep -- `as.polygons()` dissolves borders by default.

Reply:
Excellent, I haven't been aware of this, and I think it was different with the raster command back then. Anyways, thanks a lot for noting, makes the code more concise!